src/test/java/org/cactoos/io/TeeInputFromBytesTest.java

```diff
@@ -51,40 +51,43 @@ public void copiesFromBytesToPath() throws IOException {
         final String message =
             "Hello, товарищ path äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Bytes to path test - Message content is required",
             new TeeInput(
                 new BytesOf(message),
                 output.toPath()
             ),
             new InputHasContent(message)
-        );
+        ).affirm();
```

Comment:
@marceloamadeu reading, I realise this test is incorrect: it does not test that the path actually has the content, just that `TeeInput`, as an `Input` has the content: can you add a `todo` for fixing this problem for all `TeeInput` tests (this file and others test files) where it makes sense?

Reply:
@victornoel Done ... please check.